Readme modify (#12)
taichi-dev/Taichi-UnityExample: Build.md

> ## Run the demo
> This demo may not run directly on your device. So we have listed some of the problems you may encounter here.

Reviewer: ```suggestion
# How to build Taichi Unity examples

Because Taichi AOT is in active development, the Unity examples might suffer from unintentional breakage. This documentation helps you to recover from the incidents if we cannot immediately address your issue in the first place.

```

> - **If you get the follow popup when trying to run:**

Reviewer: ```suggestion
## Assertion failed in `serialization.h`
```

<div align = center>
<img src = "img/dll_crash.png" width="70%">
</div>

> This means that you need to regenerate the `taichi_c_api.dll` and `taichi_unity.dll` files and put them in the `Assets/Plugins/X86_64`.

Reviewer: ```suggestion
It means that `taichi_c_api.dll` and `taichi_unity.dll` in `Assets/Plugins/X86_64` are incompatible with the AOT modules in `Assets/Resources/TaichiModules`. You should recompile the AOT modules with programs in `scripts`, and update the DLLs from a same version of Taichi.
```

> Among them, the generation of taichi_c_api.dll is completed during the source code compilation process of taichi. if you can't find `c_api` folder under `.\taichi\_skbuild\win-amd64-3.X\cmake-install`. You should set the H_C_API option to ON in the CmakeCache.txt file under the `.\taichi\_skbuild\win-amd64-3.X\cmake-build`. 

Reviewer: ```suggestion
You can find `taichi_c_api.dll` in [a nightly Python wheel](https://pypi.org/project/taichi-nightly/). Or you can build it from source following the steps in [Developer Installation](https://docs.taichi-lang.org/docs/dev_install) with CMake definition `TI_WITH_C_API=ON`. The Taichi Runtime C-API library can be found in `taichi/_skbuild/<platform>/cmake-install/c_api`.

To ensure you are building with `TI_WITH_C_API`, please lookup the CMake options in `taichi/_skbuild/<platform>/cmake-build/CMakeCache.txt`.
```

<div align = center>
<img src = "img/capi_compile.png" width="70%">
</div>

After recompiling, you can find the dll file in the c_api folder mentioned earlier.
<div align = center>
<img src = "img/capi_cll.png" width="70%">
</div>

You can get `taichi_unity.dll` from [taichi-dev/taichi-unity2](https://github.com/taichi-dev/taichi-unity2) in a similar way.

In this way, our demo will no longer crash :).

> - **If you get the follow Exception when trying to run:**

Reviewer: ```suggestion
## InvalidOperationException: Ignored launch because kernel handle is null
```

<div align = center>
<img src = "img/exception.png" width="70%">
</div>

> This means that we need to regenerate the compute graph.

Reviewer: It's rooted from the same compatibility issue as the one above.  You should recompile the AOT modules with programs in `scripts`, and update the DLLs from a same version of Taichi.
